roles/storage-cns/tasks/deprovision.yml

```diff
@@ -5,10 +5,13 @@
         and admin_user is defined
         and admin_password is defined
 
+- set_fact:
+    heketi_route: unused
```

Comment:
Why do you set this here? The only thing I see for this is to have a dummy value if for some reason no route can be found, but then why even run this role? At that point it should just fail.

Reply:
It's needed when generating the storage class deployment yaml from the template.  When removing the storage class we don't need the actual value of the route so there is no reason to try and look it up.

Reply:
I see that you're using `kubectl delete -f`, thus you need to render the `storage-cns.yml` template. However, if `heketi_route` doesn't count towards validating the identity of the resource to be deleted, why even bother rendering the template? Just do `kubectl delete sc kubevirt --ignore-not-found`, since the StorageClass is supposed to always have the same name anyway.

Reply:
I like to use the template method because if we add things to it (like the glusterfile provisioner) we don't need to add any new deprovision ansible tasks.

Reply:
There's something about not using set_fact if you can help it... but I don't remember what it is. :) I'll approve this for now and if I come up with a good reason I'll submit another PR to fix it.